src/Sylius/Bundle/OrderBundle/Controller/OrderController.php

```diff
@@ -183,6 +180,73 @@ public function clearAction(Request $request)
     }
 
     /**
+     * @param Request $request
+     *
+     * @return Response
+     */
+    public function completeAction(Request $request)
```

Comment:
IMO.. not, because this method is for an order entity and I do not think that there is a need to create another controller only for one method. What is more, we have in this controller also methods for an order in state cart, so if we want to extract methods, we could also create `CartController`, but I guess that we do not want that.